src/components/Input.js

```diff
@@ -0,0 +1,27 @@
+import { useState } from "react";
+import {
+  InputContainer,
+  Label,
+  Textarea,
+} from "/src/components/styles/InputStyle.js";
+
+export default function Input({ label, name, placeholder }) {
+  const [isFilled, setIsFilled] = useState(false);
+
+  const handleBlur = (event) => {
+    setIsFilled(event.target.value !== "");
+  };
```

Comment:
It would be nice to have consistent function definition, only here i found `Arrow Function`.